tests/Avalonia.Controls.UnitTests/ListBoxTests.cs

```diff
@@ -407,6 +407,53 @@ public void Clicking_Item_Should_Raise_BringIntoView_For_Correct_Control()
             Assert.Equal(1, raised);
         }
 
+        [Fact]
+        public void Adding_And_Selecting_Item_With_AutoScrollToSelectedItem_Should_NotHide_FirstItem()
+        {
+            using (UnitTestApplication.Start(TestServices.StyledWindow))
+            {
+                var items = new AvaloniaList<string>();
+
+                var wnd = new Window() { Width = 100, Height = 100, IsVisible = true };
```

Comment:
For future reference, it's often more convenient to use `TestRoot` instead of `Window` in unit tests, but if `Window` works well then 👍 .

Reply:
will fix it